### PR TITLE
Improve the automatic rollback documentation on our docs

### PR DESCRIPTION
**Pull request by georgiats**
This is a PR to add some extra info about the automatic rollback to our docs

### CODE REVIEW

**Review by vipulgupta2048 — COMMENTED** [review]
@georgiats Thanks for opening the PR, you seem to have missed the commit prefix to signify the change-type for your PR. Please read https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md#creating-commits-in-line-with-semantic-versioning and let us know if you have any further questions. Thanks again.

**Comment by georgiats** [comment]
@balena-ci rebase

**Comment by georgiats** [comment]
@balena-ci rebase

File: pages/reference/OS/updates/update-process.md [hunk 1]
```diff
@@ -58,6 +58,11 @@ If the fingerprint check succeeds, the updater will use **Docker** to pull the t
 
 From here, the original updater script takes back control, and finishes up any other changes required by the update. When all this is done, the device is rebooted into the new system.
 
+## Automatic rollbacks
+Rollbacks is a framework designed to automatically roll back the OS update in case something goes wrong.
+
+There are two rollback mechanisms in the OS, covering different update failure modes: one based on health checks, the rollback-health, and another recognizing if the new system is unbootable for some reason, the rollback-altboot. Their detailed operations are explained [here].
```
Comment (vipulgupta2048) [inline pages/reference/OS/updates/update-process.md:64]:
```suggestion
There are two rollback mechanisms in the OS, covering different update failure modes. One based on health checks called the `rollback-health`, and the recognizing if the new system is unbootable for some reason, the `rollback-altboot`. Learn more about these [rollback mechanisms][rollback-mechanism].
```

File: pages/reference/OS/updates/update-process.md [hunk 2]
```diff
@@ -58,6 +58,11 @@ If the fingerprint check succeeds, the updater will use **Docker** to pull the t
...
+
 [changes]:/reference/OS/updates/migrate-to-2.0/
 [wifi-connect]:{{ $links.githubMain }}/wifi-connect
-[automatic-rollbacks]:{{ $links.githubOS }}/meta-balena/blob/development/docs/rollbacks.md
+[here]:{{ $links.githubOS }}/meta-balena/blob/master/docs/rollbacks.md
```
Comment (vipulgupta2048) [inline pages/reference/OS/updates/update-process.md:68]:
```suggestion
[rollback-mechanism]:{{ $links.githubOS }}/meta-balena/blob/master/docs/rollbacks.md
```

File: pages/reference/OS/updates/update-process.md (hunk 1 above)
Comment (vipulgupta2048) [reply pages/reference/OS/updates/update-process.md:64]: Learn more about adding links to docs the right way here https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md#adding-links-to-the-docs

**Review by vipulgupta2048 — APPROVED** [review]

**Pull request merged**